include/openenclave/bits/evidence.h

```diff
@@ -62,29 +56,88 @@ typedef struct _oe_claim
     size_t value_size;
 } oe_claim_t;
 
-/**
+/*
  * Claims that are known to OE that every attestation plugin should output.
  */
+
+/**
+ * Claims version.
+ */
 #define OE_CLAIM_ID_VERSION "id_version"
+
+/**
+ * Security version of the enclave. (SVN for SGX).
+ */
 #define OE_CLAIM_SECURITY_VERSION "security_version"
+
+/**
+ * Attributes flags for the evidence.
+ */
 #define OE_CLAIM_ATTRIBUTES "attributes"
+
+/**
+ * The unique ID for the enclave (MRENCLAVE for SGX).
+ */
 #define OE_CLAIM_UNIQUE_ID "unique_id"
+
+/**
+ * The signer ID for the enclave (MRSIGNER for SGX).
+ */
 #define OE_CLAIM_SIGNER_ID "signer_id"
+
+/**
+ * The product ID for the enclave (ISVPRODID for SGX).
+ */
 #define OE_CLAIM_PRODUCT_ID "product_id"
+
+/**
+ * The format id of the evidence.
+ */
 #define OE_CLAIM_FORMAT_UUID "format_uuid"
+
+/**
+ * @cond DEV
+ *
+ */
+
 #define OE_REQUIRED_CLAIMS_COUNT 7
+
 // This array is needed for tests
 extern const char* OE_REQUIRED_CLAIMS[OE_REQUIRED_CLAIMS_COUNT];
 
 /**
+ * @endcond
+ *
+ */
+
+/*
  * Additional optional claims that are known to OE that plugins can output.
  */
+
+/**
+ * Overall datetime from which the evidence and endorsements are valid.
+ */
 #define OE_CLAIM_VALIDITY_FROM "validity_from"
+
+/**
+ * Overall datetime at which the evidence and endorsements expire.
+ */
 #define OE_CLAIM_VALIDITY_UNTIL "validity_until"
+
+/**
+ * @cond DEV
+ *
+ */
+
 #define OE_OPTIONAL_CLAIMS_COUNT 2
 // This array is needed for tests
 extern const char* OE_OPTIONAL_CLAIMS[OE_OPTIONAL_CLAIMS_COUNT];
```

Comment:
I don't think test-only definitions should be in public headers. I did not find any code using `OE_OPTIONAL_CLAIMS[]`. Should it be removed?

Reply:
OE_OPTIONAL_CLAIMS is DOXYGEN-commented out (by @cond DEV).

Reply:
Regardless whether DOXYGEN removes it or not, I don't think public API header files should include test-only definitions.